packages/documentation/copy/en/modules-reference/guides/Choosing Compiler Options.md

```diff
@@ -184,7 +184,9 @@ Let’s examine why we picked each of these settings:
 - **`verbatimModuleSyntax: true`**. This setting protects against a few module-related pitfalls that can cause problems for library consumers. First, it prevents writing any import statements that could be interpreted ambiguously based on the user’s value of `esModuleInterop` or `allowSyntheticDefaultImports`. Previously, it was often suggested that libraries compile without `esModuleInterop`, since its use in libraries could force users to adopt it too. However, it’s also possible to write imports that only work _without_ `esModuleInterop`, so neither value for the setting guarantees portability for libraries. `verbatimModuleSyntax` does provide such a guarantee.[^1] Second, it prevents the use of `export default` in modules that will be emitted as CommonJS, which can require bundler users and Node.js ESM users to consume the module differently. See the appendix on [ESM/CJS Interop](/docs/handbook/modules/appendices/esm-cjs-interop.html#library-code-needs-special-considerations) for more details.
 - **`declaration: true`** emits type declaration files alongside the output JavaScript. This is needed for consumers of the library to have any type information.
-- **`sourceMap: true`** and **`declarationMap: true`** emit source maps for the output JavaScript and type declaration files, respectively. These are only useful if the library also ships its source (`.ts`) files. By shipping source maps and source files, consumers of the library will be able to debug the library code somewhat more easily. By shipping declaration maps and source files, consumers will be able to see the original TypeScript sources when they run Go To Definition on imports from the libraries. Both of these represent a tradeoff between developer experience and library size, so it’s up to you whether to include them.
+- **`sourceMap: true`** emits source maps for the output JavaScript files. By shipping source maps, when consumers of the library execute the code with source maps enabled (e.g. via [`node --enable-source-maps`](https://nodejs.org/docs/latest-v18.x/api/cli.html#--enable-source-maps)), stack traces of errors will point to the original source TypeScript files (instead of the emitted output JavaScript files). When the source files are also shipped with the library, source maps allow consumers of the library to debug the library code somewhat more easily.
+- **`declarationMap: true`** emits source maps for the output type declaration files. By shipping declaration maps and source files, consumers will be able to see the original TypeScript sources when they run Go To Definition on imports from the libraries.
+- Note that **`sourceMap: true`** and **`declarationMap: true`** represent a tradeoff between developer experience and library size, so it’s up to you whether to include them.
```

Comment:
The pattern here has been for each bullet to be about a specific compiler option, but this one is clarifying the previous lines. I'm not sure how to do better, though.